tasks/gems.yml

```diff
@@ -1,6 +1,7 @@
 ---
 
-- set_fact:
+- name: packages > Set embedded ruby
+  set_fact:
     _manala_sensu_embedded_ruby: "{{ manala_sensu_config.EMBEDDED_RUBY|default(false) }}"
```

Comment:
1) Please two underscores to prefix internal variables
2) Do we really need this set_fact ? Can't we use directly the variable value ?